<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IMServerClass</class>
 <widget class="QWidget" name="IMServerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IMServer</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>643</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>监听地址</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboIP">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>0</height>
       </size>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="duplicatesEnabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>127.0.0.1</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>监听端口</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinPort">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>65535</number>
      </property>
      <property name="value">
       <number>1200</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnStart">
      <property name="text">
       <string>开启监听</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnClose">
      <property name="text">
       <string>关闭监听</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnClear">
      <property name="text">
       <string>清空文本框</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>641</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="editFilePath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSelectFile">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSendFile">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>发送文件</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editUserID">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string extracomment="用户账号"/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="IMServer.qrc"/>
 </resources>
 <connections/>
</ui>
